packages/headings/src/sass/_module.scss

```diff
@@ -26,7 +26,7 @@
 	.uikit-display-#{$i} {
 		font-family: $uikit-font;
 		display: block;
-		line-height: $uikit-leading; //1.142857143;//1.4 from 16 (default: core)
+		line-height: ($uikit-leading * 0.875); //1.4 from 16 (default: core)
```

Comment:
I’m happy to go down either path. A `leading()` function would be super easy to do — I’m happy to pick this up and make this edit if you like @dominikwilkowski ?